python/samples/getting_started_with_agents/chat_completion/step10_chat_completion_agent_structured_outputs.py

```diff
@@ -58,7 +58,7 @@ async def main():
     # 4. Invoke the agent for a response
     response = await agent.get_response(message=USER_INPUT, thread=thread)
     # 5. Validate the response and print the structured output
-    reasoned_result = Reasoning.model_validate(json.loads(response.content))
+    reasoned_result = Reasoning.model_validate(json.loads(response.message.content))
```

Comment:
That could also be, as an FYI:

```suggestion
    reasoned_result = Reasoning.model_validate(json.loads(str(response.content)))
```